### DISCUSSION TITLE
Part5. CH01_08 질문입니다.

### DISCUSSION THREAD

**Discussion by industrial-engineering-person** [tedilabs/fastcampus-devops#64] [Q&A] [answered]
환경은 wsl의 ubuntu입니다.

문제가 생겼는데 스스로 해결하지 못하여 질문 드립니다 ㅠ
코드에서 수정한건 key_name을 제가 aws에서 생성한 이름으로 변경하였고 
aws connection에 ssh접근을위해 private_key 속성을 추가해주었습니다. 이렇게 ssh접속은 되었는데 다음에러가 발생했습니다.

에러는 다음과 같습니다
![image](https://user-images.githubusercontent.com/93564273/166100332-8b66b9d4-924b-46f2-814a-d51222de7175.png)

====> 어쩌다보니 해결이 되었습니다.. ㅎㅎ
윈도우 사용하다보니  \r\n줄 끝이 있어서 .Unix.sh로 수정하여 해결하였습니다.

혹시 다음 저와 같은 문제가 있으신분은
https://stackoverflow.com/questions/29045140/env-bash-r-no-such-file-or-directory 
위 링크를 참고하시면 좋을것 같습니다.

**Comment by posquit0** [comment, marked as answer]
답변이 늦었는데 이미 해결하셨군요!
근본적인 원인은 윈도우와 리눅스가 사용하는 개행문자(New Line)가 서로 다르다는 것입니다. 해당 파일을 다운로드 받거나 수정하는 과정에서 개행문자가 변경되어, EC2 리눅스 환경에서 개행문자 해석 오류가 발생한 것으로 보입니다.

다음에 또 질문이 있다면 찾아주세요. :)
